cookbooks/aws-parallelcluster-platform/spec/unit/recipes/sudo_config_spec.rb

```diff
@@ -1,14 +1,15 @@
 require 'spec_helper'
 
-describe 'aws-parallelcluster-config::sudo' do
+describe 'aws-parallelcluster-platform::sudo_config' do
   for_all_oses do |platform, version|
     context "on #{platform}#{version}" do
       cached(:chef_run) do
-        ChefSpec::Runner.new(platform: platform, version: version).converge(described_recipe)
+        runner(platform: platform, version: version).converge(described_recipe)
       end
 
       it 'creates the sudoers template with the correct attributes' do
         is_expected.to create_template('/etc/sudoers.d/99-parallelcluster-user-tty').with(
+          # source: 'base/99-parallelcluster-user-tty.erb',
```

Comment:
why is this commented out?

Reply:
My bad I added it and while testing I commented it out.